e2e/libs/workload/workload.py

```
@@ -223,6 +223,21 @@ def wait_for_workload_pods_running(workload_name, namespace="default"):
+    retry_count, retry_interval = get_retry_count_and_interval()
+    for i in range(retry_count):
+        pods = get_workload_pods(workload_name, namespace=namespace)
+        if len(pods) > 0:
+            for pod in pods:
+                if pod.status.phase == "Pending":
+                    return
+
+        logging(f"Waiting for {workload_name} pods {[pod.metadata.name for pod in pods]} container creating, retry ({i}) ...")
+        time.sleep(retry_interval)
+
+    assert False, f"Timeout waiting for {workload_name} pods container creating"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)